src/quo/components/buttons/button/properties.cljs

```diff
@@ -5,10 +5,10 @@
 (def backgrounds #{:photo :blur})
 
 (defn- custom-color-type
-  [customization-color icon-only? theme]
+  [customization-color icon-only?]
   {:icon-color                  colors/white-opa-70
    :label-color                 colors/white
-   :background-color            (colors/resolve-color customization-color theme)
+   :background-color            (colors/custom-color customization-color 50)
```

Comment:
I'm not sure how to replace this deprecated function. @J-Son89 If you have some insights please let me know.

Reply:
hmm I don't think this is the issue, what you had before is correct actually

Reply:
`:background-color (colors/resolve-color customization-color theme)` should work

Reply:
in fact, you are kind of hard coding "light" theme here so this can't be right

Reply:
My bad! 🙃 
This is actually correct. 
We should hardcode this as theme light here `(colors/resolve-color customization-color :light)`

The reason we do this is because we need the button to handle both customization-colors and community colors. However with community colors we only receive a light themed version of the community color (i,e a `-50` variant of that color). For that reason we use a shadow overlay to get the `-60` variant.

Reply:
Hi @J-Son89,
I am still little confused. So if for communities we are only receiving light themed version and use shadow overlay for 60 variants, what about other places?

Shouldn't button accept theme value, and we pass/hard coding at use case?

Reply:
the theming is used but it is used in the overlay which is in the button

Reply:
Like in reported issue, we use/hard code dark theme

Reply:
more details here:
https://github.com/status-im/status-mobile/pull/16536

Reply:
<img width="345" alt="Screenshot 2024-02-09 at 13 15 06" src="https://github.com/status-im/status-mobile/assets/22799766/39b0dfbc-97d3-4345-a1f6-a377a5ec2632">

Reply:
<img width="370" alt="Screenshot 2024-02-09 at 15 01 43" src="https://github.com/status-im/status-mobile/assets/22799766/8be2551a-fa9a-4a78-b4df-601da97ea824">

Reply:
Thank you for links, now I see how theme is used.

Reply:
Thanks @Parveshdhull, let me know if there's any further doubts! 
I think it's best @ajayesivan adds a comment to that line about this and if you can take a look to confirm it's clear enough that would be very helpful :)

Reply:
> I think it's best @ajayesivan adds a comment to that line

or on the top of `custom-color-type` function, maybe something like

`;; Note: We are hardcoding light theme as background here and then applying overlay when needed (for dark themes and pressed states). We do this because for communities we only have one color (light theme version).`

Reply:
yep perfect, and if we can link this issue too that might help - https://github.com/status-im/status-mobile/issues/16396